deploy-board/deploy_board/static/js/components/capacitycomponents.js

```diff
@@ -248,6 +265,8 @@ Vue.component("static-capacity-config", {
             } else {
                 this.showSizeError = false;
             }
+            this.imbalanceWarning = checkImbalance(this.placements, (sizeIncrease / this.placements.length).toFixed());
```

Comment:
This seems good.

I am also wondering if we could/should sum the remaining capacity in the placements from the information available "remaining capacity" for each placement? 

On one hand, the capacity is available, OTOH it can become unbalanced. In this sense, maybe the increase should follow a different calculation.

What if instead we use min() of list of remaining capacity per AZ, so basically sum remaining capacity of subnets in each AZ. This could be a followup ticket or something, the PR is already pretty large.

Reply:
What if we just move this calculation into a function for now and ticket the improvement?